monkey/monkey_island/cc/repository/i_credentials_repository.py

```diff
@@ -0,0 +1,81 @@
+from abc import ABC
+from typing import Any, Sequence
+
+from monkey.cc.models import ConfiguredCredentials, StolenCredentials
+
+
+class ICredentialsRepository(ABC):
+    def get_configured_credentials(self) -> Sequence[ConfiguredCredentials]:
```

Comment:
So if we add, lets say, credentials from wordlist we'll have to change the whole interface and add 3 methods for each new credential origin? I think there can easily be one method `get_credentials(self, origin=None)` to get all, configured or stolen.

Reply:
Probably credentials from wordlist are configured credentials but I get what you are saying and I agree on that. The interface as it is now is complicated.

Reply:
Credentials from a wordlist would be "configured". Either credentials are input to the simulation (configured) or the agent takes some action to find new credentials (stolen). Is there a possible third category?